extensions/cornerstone/src/init.js

```diff
@@ -78,6 +78,16 @@ export default function init(configuration = {}) {
   ];
 
   tools.forEach(tool => csTools.addTool(tool));
+
+  const BaseAnnotationTool = csTools.importInternal('base/BaseAnnotationTool');
+  tools.forEach(tool => {
+    if (tool.prototype instanceof BaseAnnotationTool) {
+      // BaseAnnotationTool would likely come from csTools lib exports
+      const toolName = new tool().name;
```

Comment:
I get your concern that it feels a bit wonky.
The rationale that they are not static properites of the class is that one may make multiple tools by just passing different configuration. 

The default tool names are just the classnames with the suffix removed `Tool`, so you can do an `Object.keys` to get the name, and `split` off the Tool.

Reply:
Note: Perhaps it'd be good in CST if we expose a default name explictly on the class, and then use this in the constructor if the default isn't changed. Its all implicit right now.

Reply:
Heh. Gustavo and I worked together on this bit. At the time, I said it was "good enough", but there's a better solution out there. Cornerstone Tools tracks `GlobalTools` and `tools`. If we updated the "History Tracking" changes we discussed earlier this year (RE: @JamesAPetts), then you could just iterate the list of `GlobalTools` and use the names defined there instead of using this hack.

(Which would fail if any tool specified a custom name when initializing, this just grabs the default name.)

Reply:
> this just grabs the default name,

So right now we can do get the default name also by taking the class and removing `Tool` from the class name. That still feels better than instantiating a tool for no reason, no?

Reply:
For CornerstoneTools, that's fine. I'm at a bit of a loss as we open up the configuration to allow registering/adding custom tools.

Reply:
Eh, I guess you are right. This is "better" for now.
The upgrade required for CST is super trivial though, just need to go through each core tool and update the default name in the constructor to be taken from a static `BlahTool.DEFAULT_TOOLNAME` prop. , could lump it as part of this PR's work. Then third party tools can do the same.

Reply:
The issue reported in this comment related with `data.length.toFixed is not a function` should be fixed by the latest version of `dcmjs`. Thanks for helping me on that @JamesAPetts 

The other issue reported, related with the way we get the tool name, I've spoke with Danny and this is something that we are going to change in the future, in order to catch the actual name that each tool was instantiated with and not the default name in the tool. In the near future, this changes would be unacessary in our point of view. 
We can discuss this later as well.

Reply:
Yeah this certainly isn't blocking for now, but something we can clean up in the future. We do only do this once, so its not a huge overhead.